### ISSUE TITLE
Node error on 9.0.0, invalid flag --use-openssl-ca

### ISSUE THREAD

**Issue by jerbob92** [mend/renovate-ce-ee#622]
I'm getting the following error after the upgrade to 9.0.0:

```
➤ YN0001: │ Error [ERR_WORKER_INVALID_EXEC_ARGV]: @babel/helper-validator-identifier@npm:7.25.9: Initiated Worker with invalid execArgv flags: --use-openssl-ca
    at new Worker (node:internal/worker:222:13)
    at B2.createWorker (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:192:53159)
    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:192:53649
    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3740
    at new Promise (<anonymous>)
    at Mz.exports (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3722)
    at a (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3962)
    at n (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:4011)
    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:4081
    at new Promise (<anonymous>)
```

It looks like the options that are being used are not compatible with the Node version used in the base image of 9.0.0.

**Comment by rarkins** [comment]
Please provide extended logs, e.g. the 100 lines at DEBUG level before this error message

**Comment by jerbob92** [comment]
Anything specific you are looking for? A lot of stuff happening in the logs for a lot off different repo's, hard to give the raw logs for this specific issue and without exposing sensitive data.

I do have this for you:

```
        "err": {
          "cmd": "/bin/sh -c yarn install --mode=update-lockfile",
          "stderr": "",
          "stdout": "➤ YN0000: · Yarn 4.5.1\n➤ YN0000: ┌ Resolution step\n➤ YN0085: │ + npm-run-all2@npm:5.0.2, @babel/code-frame@npm:7.26.2, @babel/helper-validator-identifier@npm:7.25.9, @types/normalize-package-data@npm:2.4.4, and 8 more.\n➤ YN0085: │ - array-buffer-byte-length@npm:1.0.0, arraybuffer.prototype.slice@npm:1.0.1, available-typed-arrays@npm:1.0.5, call-bind@npm:1.0.2, cross-spawn@npm:6.0.5, and 63 more.\n➤ YN0000: └ Completed in 0s 756ms\n➤ YN0000: ┌ Post-resolution validation\n➤ YN0060: │ date-fns is listed by your project with version 4.1.0 (p5c432), which doesn't satisfy what @date-io/date-fns requests (^3.2.0).\n➤ YN0086: │ Some peer dependencies are incorrectly met by your project; run yarn explain peer-requirements <hash> for details, where <hash> is the six-letter p-prefixed code.\n➤ YN0086: │ Some peer dependencies are incorrectly met by dependencies; run yarn explain peer-requirements for details.\n➤ YN0000: └ Completed\n➤ YN0000: ┌ Fetch step\n➤ YN0001: │ Error [ERR_WORKER_INVALID_EXEC_ARGV]: cross-spawn@npm:7.0.6: Initiated Worker with invalid execArgv flags: --use-openssl-ca\n    at new Worker (node:internal/worker:222:13)\n    at B2.createWorker (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:192:53159)\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:192:53649\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3740\n    at new Promise (<anonymous>)\n    at Mz.exports (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3722)\n    at a (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3962)\n    at n (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:4011)\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:4081\n    at new Promise (<anonymous>)\n➤ YN0001: │ Error [ERR_WORKER_INVALID_EXEC_ARGV]: type-fest@npm:0.6.0: Initiated Worker with invalid execArgv flags: --use-openssl-ca\n    at new Worker (node:internal/worker:222:13)\n    at B2.createWorker (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:192:53159)\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:192:53649\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3740\n    at new Promise (<anonymous>)\n    at Mz.exports (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3722)\n    at a (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3962)\n    at n (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:4011)\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:4081\n    at new Promise (<anonymous>)\n➤ YN0001: │ Error [ERR_WORKER_INVALID_EXEC_ARGV]: lines-and-columns@npm:1.2.4: Initiated Worker with invalid execArgv flags: --use-openssl-ca\n    at new Worker (node:internal/worker:222:13)\n    at B2.createWorker (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:192:53159)\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:192:53649\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3740\n    at new Promise (<anonymous>)\n    at Mz.exports (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3722)\n    at a (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3962)\n    at n (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:4011)\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:4081\n    at new Promise (<anonymous>)\n➤ YN0001: │ Error [ERR_WORKER_INVALID_EXEC_ARGV]: npm-run-all2@npm:5.0.2: Initiated Worker with invalid execArgv flags: --use-openssl-ca\n    at new Worker (node:internal/worker:222:13)\n    at B2.createWorker (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:192:53159)\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:192:53649\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3740\n    at new Promise (<anonymous>)\n    at Mz.exports (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3722)\n    at a (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3962)\n    at n (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:4011)\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:4081\n    at new Promise (<anonymous>)\n➤ YN0001: │ Error [ERR_WORKER_INVALID_EXEC_ARGV]: read-pkg@npm:5.2.0: Initiated Worker with invalid execArgv flags: --use-openssl-ca\n    at new Worker (node:internal/worker:222:13)\n    at B2.createWorker (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:192:53159)\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:192:53649\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3740\n    at new Promise (<anonymous>)\n    at Mz.exports (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3722)\n    at a (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3962)\n    at n (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:4011)\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:4081\n    at new Promise (<anonymous>)\n➤ YN0001: │ Error [ERR_WORKER_INVALID_EXEC_ARGV]: @babel/helper-validator-identifier@npm:7.25.9: Initiated Worker with invalid execArgv flags: --use-openssl-ca\n    at new Worker (node:internal/worker:222:13)\n    at B2.createWorker (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:192:53159)\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:192:53649\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3740\n    at new Promise (<anonymous>)\n    at Mz.exports (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3722)\n    at a (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3962)\n    at n (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:4011)\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:4081\n    at new Promise (<anonymous>)\n➤ YN0001: │ Error [ERR_WORKER_INVALID_EXEC_ARGV]: json-parse-even-better-errors@npm:2.3.1: Initiated Worker with invalid execArgv flags: --use-openssl-ca\n    at new Worker (node:internal/worker:222:13)\n    at B2.createWorker (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:192:53159)\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:192:53649\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3740\n    at new Promise (<anonymous>)\n    at Mz.exports (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3722)\n    at a (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3962)\n    at n (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:4011)\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:4081\n    at new Promise (<anonymous>)\n➤ YN0001: │ Error [ERR_WORKER_INVALID_EXEC_ARGV]: pidtree@npm:0.5.0: Initiated Worker with invalid execArgv flags: --use-openssl-ca\n    at new Worker (node:internal/worker:222:13)\n    at B2.createWorker (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:192:53159)\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:192:53649\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3740\n    at new Promise (<anonymous>)\n    at Mz.exports (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3722)\n    at a (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3962)\n    at n (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:4011)\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:4081\n    at new Promise (<anonymous>)\n➤ YN0001: │ Error [ERR_WORKER_INVALID_EXEC_ARGV]: parse-json@npm:5.2.0: Initiated Worker with invalid execArgv flags: --use-openssl-ca\n    at new Worker (node:internal/worker:222:13)\n    at B2.createWorker (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:192:53159)\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:192:53649\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3740\n    at new Promise (<anonymous>)\n    at Mz.exports (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3722)\n    at a (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3962)\n    at n (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:4011)\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:4081\n    at new Promise (<anonymous>)\n➤ YN0001: │ Error [ERR_WORKER_INVALID_EXEC_ARGV]: @babel/code-frame@npm:7.26.2: Initiated Worker with invalid execArgv flags: --use-openssl-ca\n    at new Worker (node:internal/worker:222:13)\n    at B2.createWorker (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:192:53159)\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:192:53649\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3740\n    at new Promise (<anonymous>)\n    at Mz.exports (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3722)\n    at a (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3962)\n    at n (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:4011)\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:4081\n    at new Promise (<anonymous>)\n➤ YN0001: │ Error [ERR_WORKER_INVALID_EXEC_ARGV]: ansi-styles@npm:5.2.0: Initiated Worker with invalid execArgv flags: --use-openssl-ca\n    at new Worker (node:internal/worker:222:13)\n    at B2.createWorker (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:192:53159)\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:192:53649\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3740\n    at new Promise (<anonymous>)\n    at Mz.exports (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3722)\n    at a (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3962)\n    at n (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:4011)\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:4081\n    at new Promise (<anonymous>)\n➤ YN0001: │ Error [ERR_WORKER_INVALID_EXEC_ARGV]: @types/normalize-package-data@npm:2.4.4: Initiated Worker with invalid execArgv flags: --use-openssl-ca\n    at new Worker (node:internal/worker:222:13)\n    at B2.createWorker (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:192:53159)\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:192:53649\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3740\n    at new Promise (<anonymous>)\n    at Mz.exports (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3722)\n    at a (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3962)\n    at n (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:4011)\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:4081\n    at new Promise (<anonymous>)\n➤ YN0001: │ Error [ERR_WORKER_INVALID_EXEC_ARGV]: @rollup/rollup-linux-x64-gnu@npm:4.24.0: Initiated Worker with invalid execArgv flags: --use-openssl-ca\n    at new Worker (node:internal/worker:222:13)\n    at B2.createWorker (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:192:53159)\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:192:53649\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3740\n    at new Promise (<anonymous>)\n    at Mz.exports (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3722)\n    at a (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3962)\n    at n (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:4011)\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:4081\n    at new Promise (<anonymous>)\n➤ YN0001: │ Error [ERR_WORKER_INVALID_EXEC_ARGV]: @esbuild/linux-x64@npm:0.21.5: Initiated Worker with invalid execArgv flags: --use-openssl-ca\n    at new Worker (node:internal/worker:222:13)\n    at B2.createWorker (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:192:53159)\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:192:53649\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3740\n    at new Promise (<anonymous>)\n    at Mz.exports (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3722)\n    at a (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3962)\n    at n (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:4011)\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:4081\n    at new Promise (<anonymous>)\n➤ YN0001: │ Error [ERR_WORKER_INVALID_EXEC_ARGV]: @sentry/cli-linux-x64@npm:2.36.6: Initiated Worker with invalid execArgv flags: --use-openssl-ca\n    at new Worker (node:internal/worker:222:13)\n    at B2.createWorker (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:192:53159)\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:192:53649\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3740\n    at new Promise (<anonymous>)\n    at Mz.exports (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3722)\n    at a (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:3962)\n    at n (/opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:4011)\n    at /opt/containerbase/tools/yarn/4.5.1/22.11.0/node_modules/@yarnpkg/cli-dist/bin/yarn.js:132:4081\n    at new Promise (<anonymous>)\n➤ YN0000: └ Completed in 0s 266ms\n➤ YN0000: · Failed with errors in 1s 31ms\n",
          "options": {
            "cwd": "/tmp/renovate/repos/gitlab/repo",
            "encoding": "utf-8",
            "env": {
              "NPM_CONFIG_CACHE": "/tmp/renovate/cache/others/npm",
              "npm_config_store": "/tmp/renovate/cache/others/pnpm",
              "CI": "true",
              "YARN_ENABLE_IMMUTABLE_INSTALLS": "false",
              "YARN_HTTP_TIMEOUT": "100000",
              "YARN_GLOBAL_FOLDER": "/tmp/renovate/cache/others/berry",
              "YARN_ENABLE_GLOBAL_CACHE": "1",
              "HOME": "/home/ubuntu",
              "PATH": "/home/ubuntu/.local/bin:/home/ubuntu/bin:/home/ubuntu/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
              "LC_ALL": "C.UTF-8",
              "LANG": "C.UTF-8"
            },
            "maxBuffer": 10485760,
            "timeout": 900000
          },
          "exitCode": 1,
          "name": "ExecError",
          "message": "Command failed: yarn install --mode=update-lockfile\n",
          "stack": "ExecError: Command failed: yarn install --mode=update-lockfile\n\n    at ChildProcess.<anonymous> (/usr/src/app/node_modules/renovate/lib/util/exec/common.ts:101:11)\n    at ChildProcess.emit (node:events:530:35)\n    at ChildProcess.emit (node:domain:489:12)\n    at Process.ChildProcess._handle.onexit (node:internal/child_process:293:12)"
        },

```

**Comment by rarkins** [comment]
We need every single debug log message for that branch prior, or can't help you

**Comment by jerbob92** [comment]
Alright

**Comment by rarkins** [comment]
Please don't raise issues in future if you can't provide logs. It's just a waste of everyone's time.

**Comment by jerbob92** [comment]
I can provide logs, but you're just being unreasonable. Please write proper issue raising instructions.

**Comment by rarkins** [comment]
If you are going to be rude then you're not welcome to create any future issues at all.

**Comment by jerbob92** [comment]
Fixed this error (and `Unknown Syntax Error: Unsupported option name ("--ignore-platform")`) by setting `RENOVATE_BINARY_SOURCE` to `install`.

**Comment by rarkins** [comment]
Thank you for sharing your solution. Presumably you are using the `-full` image/tag, which:
- Has each language/package manager preinstalled
- Does not attempt to dynamically install any versions at runtime (binarySource=global)
- Is more susceptible to language or package manager incompatibility (because the image has only one version of each installed)

The non-full tag has binarySource=install by default

**Comment by jerbob92** [comment]
Yeah that's correct. 

I think that `binarySource=global` would work fine for most package managers/languages but Node/NPM/Yarn is a tricky one, so sticking to `install`/non-full tag for now.

Is there any functional difference between -full and non-full when using `binarySource=install`? (so was -full pre 9.0.0 actually different from non-full pre 9.0.0?).

**Comment by rarkins** [comment]
`-full` was always meant to default to binarySource=global, but somehow regressed some time ago so we decided to fix it back again in v9.0.0. There are plenty of companies who can't allow runtime install so -full is their only choice.

The only advantage of -full for binarySource=install is that the ~latest versions of everything are preinstalled at build time, so you don't have to install _everything_ at runtime. You might get a little boost on first run if plenty of the preinstalled managers are still the latest version. On the other hand the image size is much increased, and you probably won't use every single manager, so there's a lot of wasted download there for most.

**Comment by jerbob92** [comment]
Thanks, makes sense. Will move to the non-full version on the next update.
